src/setup-beam.js

```diff
@@ -209,11 +208,7 @@ async function getOTPVersions(osVersion) {
       .split('\n')
       .forEach((line) => {
         const otpMatch = line.match(/^(OTP-)?([^ ]+)/)
-
```

Comment:
This was moved to `getVersionFromSpec` to centralise version management/handling.